components/grid/export/overview.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Overview
+page_title: Grid - Export Overview
+description: Export basics for the Grid for Blazor.
+slug: grid-export-overview
+tags: telerik, blazor, grid, export
+published: True
+position: 1
+---
+
+# Blazor Grid Export
+
+The Grid for Blazor provides a built-in functionality to export the data to:
+* [CSV](slug:grid-export-csv)
+* [Excel](slug:grid-export-excel)
+* [PDF](slug:grid-export-pdf)
+
+Before proceeding to the dedicated export articles, ensure you are familiar with the following specifics:
+
+## How the Export Works
+
+The Grid export feature has the following specifics:
+
+* If the Grid is using `OnRead` and is exporting all pages, it will fire an additional `OnRead` event at the time of exporting, with a request `PageSize` of `0`. This will enable the component to obtain all data.
+* The time for export will be longer if:
+    * The Grid has a lot of records and/or 
+    * The Grid is in a Client-side Blazor (WebAssembly) where all the code runs in the browser. At the time of writing, it is considerably slower than server-side Blazor, and it only has one actual thread.
+
+>tip While the file is being generated, the UI will be unresponsive, so you may want to [show a loading sign to the user during the export process](slug:grid-kb-show-loader-while-exporting).
+
+## Requirements
+
+The Grid export feature has the following requirements:
+
+* With Server-side Blazor, the file may become larger than the default SignalR connection limit, and this can disconnect the client and result in an error. Generally, this requires quite a lot of data to happen, but you may need to increase the size limit of the connection in the `ConfigureServices` method of your `Startup.cs` file, for example:
```

Comment:
```suggestion
In server-side Blazor apps, the file may become larger than the default SignalR message size limit. This can disconnect the client and result in an error. You may need to [increase the maximum SignalR message size](slug:common-kb-increase-signalr-max-message-size).
```